Quantum-Control-Applications/Superconducting/Single-Flux-Tunable-Transmon/17_cryoscope_1ns.py

```diff
@@ -45,41 +50,78 @@
 from macros import ge_averaged_measurement
 from scipy import signal, optimize
 import matplotlib.pyplot as plt
+from scipy.integrate import simps
 
 
 ####################
 # Helper functions #
 ####################
-def exponential_decay(x, a, t):
+def exponential_decay(x, a, t, s):
     """Exponential decay defined as 1 + a * np.exp(-x / t).
 
     :param x: numpy array for the time vector in ns
     :param a: float for the exponential amplitude
     :param t: float for the exponential decay time in ns
     :return: numpy array for the exponential decay
     """
-    return 1 + a * np.exp(-x / t)
+    return s * (1 + a * np.exp(-x / t))
+
+
+def dual_exponential_decay(x, a1, t1, a2, t2, s):
+    """
+    Dual exponential decay defined as s * (1 + a1 * np.exp(-x / t1) + a2 * np.exp(-x / t2)).
+    :param x: numpy array for the time vector in ns
+    :param a1: float for the first exponential amplitude
+    :param t1: float for the first exponential decay time in ns
+    :param a2: float for the second exponential amplitude
+    :param t2: float for the second exponential decay time in ns
+    :param s: scale factor, default is 1
```

Comment:
You need to add the default value then `def dual_exponential_decay(x, a1, t1, a2, t2, s=1):`